### PR TITLE
added documentation for accepting and declining a bank account approval

### PR DESCRIPTION
**Pull request by AndyClifford**

### CODE REVIEW

File: src/api/bank-accounts/bank-account-approval-activity.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{% h4 Mandatory Fields %}
+
+|     Field      |        Type        |                             Description                              |
+| :------------- | :----------------- | :------------------------------------------------------------------- |
+| activityNumber | Number             | Unique sequential Bank Account Approval Activity number.             |
+| approvalId     | String             | The id of the associated [Bank Account Approval][].                  |
+| activityType   | String             | The type of the Bank Account Approval Activity.                      |
+| createdAt      | {% dt Timestamp %} | When the Bank Account Approval Activity was created.                 |
+| createdBy      | {% dt CRN %}       | The User or API Key that created the Bank Account Approval Activity. |
+
```
Comment (MeganSteenkamp) [inline src/api/bank-accounts/bank-account-approval-activity.md:30]: @GeekOnGadgets @ncjones I worked with Andy on this and we documented the Bank Account Approval Activity as this is what the endpoints are related to in Notion. I'm not sure if they should just be documented under the Bank Account Approval

Comment (ncjones) [reply src/api/bank-accounts/bank-account-approval-activity.md:30]: Yes, this is part of bank account approval.

File: src/api/bank-accounts/bank-account-approvals.md [hunk 2]
```diff
@@ -35,6 +35,17 @@ An `approved` Bank Account Approval is required for the funds in a [Settlement W
 | modifiedAt    | {% dt Timestamp %} | When the Bank Account Approval was updated.                                                                            |
```
Comment (ncjones) [inline src/api/bank-accounts/bank-account-approvals.md:35]: @AndyClifford @MeganSteenkamp should there be an array of activities on this approval model?

**Review by ncjones — APPROVED** [review]

File: src/api/bank-accounts/bank-account-approvals.md (hunk 2 above)
Comment (AndyClifford) [reply src/api/bank-accounts/bank-account-approvals.md:35]: yep will add that now

**Pull request merged**